src/main/fc/controlrate_profile.c

```diff
@@ -33,21 +33,34 @@
 const controlRateConfig_t *currentControlRateProfile;
 
 
-PG_REGISTER_ARRAY_WITH_RESET_FN(controlRateConfig_t, MAX_CONTROL_RATE_PROFILE_COUNT, controlRateProfiles, PG_CONTROL_RATE_PROFILES, 0);
+PG_REGISTER_ARRAY_WITH_RESET_FN(controlRateConfig_t, MAX_CONTROL_RATE_PROFILE_COUNT, controlRateProfiles, PG_CONTROL_RATE_PROFILES, 1);
 
 void pgResetFn_controlRateProfiles(controlRateConfig_t *instance)
 {
     for (int i = 0; i < MAX_CONTROL_RATE_PROFILE_COUNT; i++) {
         RESET_CONFIG(controlRateConfig_t, &instance[i],
-            .rcExpo8 = 70,
-            .thrMid8 = 50,
-            .thrExpo8 = 0,
-            .dynThrPID = 0,
-            .rcYawExpo8 = 20,
-            .tpa_breakpoint = 1500,
-            .rates[FD_ROLL] = CONTROL_RATE_CONFIG_ROLL_PITCH_RATE_DEFAULT,
-            .rates[FD_PITCH] = CONTROL_RATE_CONFIG_ROLL_PITCH_RATE_DEFAULT,
-            .rates[FD_YAW] = CONTROL_RATE_CONFIG_YAW_RATE_DEFAULT
+            .throttle = {
+                .rcMid8 = 50,
+                .rcExpo8 = 0,
+                .dynPID = 0,
+                .pa_breakpoint = 1500
+            },
+
+            .stabilized = {
+                .rcExpo8 = 70,
+                .rcYawExpo8 = 20,
+                .rates[FD_ROLL] = CONTROL_RATE_CONFIG_ROLL_PITCH_RATE_DEFAULT,
+                .rates[FD_PITCH] = CONTROL_RATE_CONFIG_ROLL_PITCH_RATE_DEFAULT,
+                .rates[FD_YAW] = CONTROL_RATE_CONFIG_YAW_RATE_DEFAULT,
+            },
+
+            .manual = {
+                .rcExpo8 = 70,
+                .rcYawExpo8 = 20,
```

Comment:
Suggest we keep manual expos at zero by default to retain 100% compatibility with old PASSTHROUGH behavior. What do you think?

Reply:
Expo is applied even in PASSTHROUGH that's why I set the default expo/yaw_expo to 70/20 to keep the old behavior but I don't mind setting them to 0 by default.

Reply:
About the MSPv2 frame there is already MSP2_COMMON_SETTING and MSP2_COMMON_SET_SETTING frame types that can be used to query/set the new settings. Do we need a new one ?

Reply:
@shellixyz Regarding expo - I see. Didn't realize that. Then it's ok.

Regarding MSPv2 - it would still be good to set this settings in a single message. Possibly a new message to handle the whole rate profile data.

Reply:
Done. New frame IDs are 0x2007 (MSP2_INAV_RATE_PROFILE) and 0x2008 (MSP2_INAV_SET_RATE_PROFILE) because I already used lower IDs for new messages in #2668.